api/v1beta1/openstackprovisionserver_types.go

```diff
@@ -26,8 +26,17 @@ type OpenStackProvisionServerSpec struct {
 	Port int `json:"port"`
 	// An optional interface to use instead of the cluster's default provisioning interface (if any)
 	Interface string `json:"interface,omitempty"`
-	// URL RHEL qcow2 image (compressed as gz, or uncompressed)
+	// URL for RHEL qcow2 image (compressed as gz, or uncompressed)
 	BaseImageURL string `json:"baseImageUrl"`
+	// Container image URL for init container that downloads the RHEL qcow2 image (baseImageUrl)
+	// +kubebuilder:default="quay.io/abays/downloader:0.0.3"
```

Comment:
should we add those images under openstack-k8s-operators ?

Reply:
Yes, let's do that.  While we're at it, I am going to rename the `downloader` image to something like `rhel-downloader` to give it more context.

Reply:
Done